### PR TITLE
Body copy changed to black.

### PR DESCRIPTION
**Pull request by taisdesouzalessa**
To check that this is working correctly please look at the subhead of the hero section and the color of the text should be set to black.

image

### CODE REVIEW

File: source/sass/global.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 body > .wrapper {
   background: $white;
+  color: $black;
```
Comment (mmmavis) [inline source/sass/global.scss]: yay @taisdesouzalessa 's first PR 🎉 

---

When you inspect the DOM element you see that text color came from CSS rule set on `body`, not `body > .wrapper`

```scss
body {
...
  color: #4d4f53;
...
}
```

screen shot 2017-04-13 at 3 07 00 pm

What we wanna do here is to change the color code to `$black`. To do this, we can simply add the following to the top of the file.

```scss
body {
  color: $black;
}
```

Comment (taisdesouzalessa) [inline source/sass/global.scss]: ...and I did my first PR wrong...hahaha. I am fixing it now.

**Comment by taisdesouzalessa** [comment]
@mmmavis made the changes as requested, please review.

**Comment by mmmavis** [comment]
Awesome work @taisdesouzalessa !!! 👍 👍 👍 Let's celebrate with some 🍰 and 🍸 for your first merged PR on this repo!

**Comment by hannahkane** [comment]
🍪  🍰  🍹 Yay Taís!

**Comment by taisdesouzalessa** [comment]
Wow, this is sooo exciting! Thanks for the celebration, team. Thanks to @flukeout for guiding me step by step too! So patient he is.

**Comment by gvn** [comment]
Nice job, @taisdesouzalessa ! You can also close #267 now if this resolved it! 👍

**Pull request merged**